tilt_modules/cert_manager/Tiltfile

```diff
@@ -26,27 +26,30 @@ spec:
 """
 
 # Deploys cert manager to your environment
-def deploy_cert_manager(registry="quay.io/jetstack", version="v1.0.1", load_to_kind=False, kind_cluster_name="kind"):
+def deploy_cert_manager(registry="quay.io/jetstack", version="v1.1.0", load_to_kind=False, kind_cluster_name="kind"):
     silent=True
 
-    # check if cert-mamager is already installed, otherwise pre-load images & apply the manifest
-    # NB. this is required until https://github.com/jetstack/cert-manager/issues/3121 is addressed otherwise
-    # when applying the manifest twice to same cluster kubectl get stuck
+    if version.startswith('v0'):
+        cert_manager_test_resources_versioned = cert_manager_test_resources.format(cert_manager_api_version='v1alpha2')
+    else:
+        cert_manager_test_resources_versioned = cert_manager_test_resources.format(cert_manager_api_version='v1')
     existsCheck = str(local("kubectl get namespaces", quiet=silent, echo_off=silent))
     if existsCheck.find("cert-manager") == -1:
-        if load_to_kind == True:
-            print("Loading images to kind")
-            # Prepull all the cert-manager images to your local environment and then load them directly into kind. This speeds up
-            # setup if you're repeatedly destroying and recreating your kind cluster, as it doesn't have to pull the images over
-            # the network each time.
-            images = ["cert-manager-controller", "cert-manager-cainjector", "cert-manager-webhook"]
-            for image in images:
-                local("docker pull {}/{}:{}".format(registry, image, version), quiet=silent, echo_off=silent)
-                local("kind load docker-image --name {} {}/{}:{}".format(kind_cluster_name, registry, image, version), quiet=silent, echo_off=silent)
+    if load_to_kind == True:
```

Comment:
I got this error when running `make tilt-up` locally:

`Error: cannot load ext://cert_manager: [...]/cluster-api-provider-azure/tilt_modules/cert_manager/Tiltfile:38:7: got if, want indent`

Reply:
It should be fixed now.